### PR TITLE
[chore] update build badge

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@
 </p>
 
 <p align="center">
-  <a href="https://circleci.com/gh/signalfx/splunk-otel-collector-chart">
-    <img alt="Build Status" src="https://img.shields.io/github/actions/workflow/status/signalfx/splunk-otel-collector-chart/lint-test.yaml?branch=main&style=for-the-badge">
+  <a href="https://github.com/signalfx/splunk-otel-collector-chart/actions/workflows/pre-commit.yaml?query=branch%3Amain">
+    <img alt="Build Status" src="https://img.shields.io/github/actions/workflow/status/signalfx/splunk-otel-collector-chart/pre-commit.yaml?branch=main&style=for-the-badge">
```
Comment (dmitryax) [inline README.md:15]: Why `pre-commit`? I think it should be `lint-test`

Comment (atoulme) [reply README.md:15]: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/.github/workflows/lint-test.yaml#L3 lint-test doesn't run against main, only pull requests. Therefore, there are no builds for the main branch and no data to display on the build page.

**Review by jvoravong — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dmitryax) [reply README.md:15]: I think we should add lint-test to main and use that one instead

Comment (atoulme) [reply README.md:15]: Sure, any reason we don't do that now?

Comment (dmitryax) [reply README.md:15]: We can do it now, sire

Comment (atoulme) [reply README.md:15]: I have opened https://github.com/signalfx/splunk-otel-collector-chart/pull/878 to follow up with lint-test, and will update this PR once it's merged.

Comment (atoulme) [reply README.md]: This is now resolved. I will merge this PR after CI passes

**Review by jinja2 — APPROVED** [review]

**Pull request merged**